posthog/models/user.py

```diff
@@ -176,30 +176,32 @@ def teams(self):
         All teams the user has access to on any organization, taking into account project based permissioning
         """
         teams = Team.objects.filter(organization__members=self)
-        if Organization.objects.filter(
-            members=self,
-            available_features__contains=[AvailableFeature.PROJECT_BASED_PERMISSIONING],
-        ).exists():
-            try:
-                from ee.models import ExplicitTeamMembership
-            except ImportError:
-                pass
-            else:
-                available_private_project_ids = ExplicitTeamMembership.objects.filter(
-                    Q(parent_membership__user=self)
-                ).values_list("team_id", flat=True)
-                organizations_where_user_is_admin = OrganizationMembership.objects.filter(
-                    user=self, level__gte=OrganizationMembership.Level.ADMIN
-                ).values_list("organization_id", flat=True)
-                # If project access control IS applicable, make sure
-                # - project doesn't have access control OR
-                # - the user has explicit access OR
-                # - the user is Admin or owner
-                teams = teams.filter(
-                    Q(access_control=False)
-                    | Q(pk__in=available_private_project_ids)
-                    | Q(organization__pk__in=organizations_where_user_is_admin)
-                )
+        org_available_product_features = Organization.objects.filter(members=self).values_list(
+            "available_product_features", flat=True
+        )
+        if org_available_product_features[0]:
```

Comment:
why is this check for the first index? should it just check len > 0 ?

Reply:
Checking for len>0 doesn't work because the org_available_product_features could be a QuerySet<None>, which has len == 1.